mmv1/products/dialogflow/EncryptionSpec.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'EncryptionSpec'
+description: |
+  Initializes a location-level encryption key specification.
+references:
+  guides:
+    'Official ES Documentation': 'https://cloud.google.com/dialogflow/es/docs'
+    'Official CX Documentation': 'https://cloud.google.com/dialogflow/cx/docs'
+  api: 'https://cloud.google.com/dialogflow/es/docs/reference/rest/v2/projects.locations.encryptionSpec'
+
+base_url: 'projects/{{project}}/locations/{{location}}/encryptionSpec'
+
+create_url: 'projects/{{project}}/locations/{{location}}/encryptionSpec:initialize'
+
+immutable: true
+
+autogen_async: true
+async:
+  actions: ['create']
+  operation:
+    base_url: '{{op_id}}'
+
+parameters:
+  - name: 'location'
+    type: String
+    required: true
+    ignore_read: true
+    description: |
+      The location in which the encryptionSpec is to be initialized.
+
+properties:
+  - name: 'encryptionSpec'
+    type: NestedObject
+    required: true
+    properties:
+      - name: 'kmsKey'
+        type: String
+        required: true
+        description: |
+          The name of customer-managed encryption key that is used to secure a resource and its sub-resources.
+          If empty, the resource is secured by the default Google encryption key.
+          Only the key in the same location as this resource is allowed to be used for encryption.
+          Format: projects/{project}/locations/{location}/keyRings/{keyRing}/cryptoKeys/{key}
+
+examples:
+  - name: "dialogflow_encryption_spec_basic"
+    primary_resource_id: "example-encryption-spec"
+    vars:
+      kms_key_name: 'kms-key'
+    test_vars_overrides:
+      kms_key_name: 'acctest.BootstrapKMSKey(t).CryptoKey.Name'
```

Comment:
Try:

```
kms_key_name: 'acctest.BootstrapKMSKeyInLocation(t, "us-central1").CryptoKey.Name'
```

The test is failing with `Please refer to https://cloud.google.com/dialogflow/cx/docs/concept/region#avail and https://cloud.google.com/dialogflow/es/docs/how/region#regions to find the correct endpoint to access resources located in 'us-central1'."` The request for the encryption spec is using `us-central`, but the KMS key's location is `global` which makes me think it's running afoul of [this limitation](https://cloud.google.com/dialogflow/cx/docs/concept/cmek#limits)